polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -109,16 +109,18 @@ export function Item({
   const textMarkup = <span className={styles.Text}>{contentMarkup}</span>;
 
   const contentElement = (
-    <HorizontalStack
-      blockAlign="center"
-      gap={polarisSummerEditions2023 ? '2' : '4'}
-      wrap={!truncate}
-    >
-      {prefixMarkup}
-      {textMarkup}
-      {badgeMarkup}
-      {suffixMarkup}
-    </HorizontalStack>
+    <Box width="100%">
```

Comment:
Does this need to be beta-flagged? e.g. `se23 ? <Box width="100%">{contentElement}</Box> : contentElement`

Reply:
This may fix the issue, but could we look into what override styles are causing this?

Looks like the added `display: flex` se23 override is what may be causing the issue.

<details>
<summary>Storybook demo</summary>

<img src="https://github.com/Shopify/polaris/assets/11774595/872dfdc1-b1c5-476d-8507-fb164c9ddee0" alt="action-list-demo">

</details>